### DISCUSSION TITLE
Download Deye / Solis Firmware

### DISCUSSION THREAD

**Discussion by PV-Joe** [davidrapan/ha-solarman#135] [General]
**The IP of the firmware update server is 47.254.36.66. It looks like behind this IP is the firmware update Server of Deye. At least I got greeted with an Nginx default page via HTTP.**

**Warning:**  
Installing any of these firmware versions may brick your system. Proceed at your own risk! Please be careful.

**Download and further information:**
https://github.com/dasrecht/deye-firmware/tree/main

**Comment by davidrapan** [comment]
@PV-Joe? Could you look for me on some of the German forums (as I can see there are plenty of them) for these specific firmware versions?:
* LSW3_32U_5406_1.06
* LSW3_32U_5406_1.07

**Reply by PV-Joe** [reply]
Hi David,

I tried but didn‘t find a download yet.
I‘m in Bulgaria at the mom and have limited access to the Internet but I keep trying.

**Comment by PV-Joe** [comment]
I think this site provides all Deye firmware versions.

https://github.com/Mights001/DeyeFirmware/wiki/

**Reply by davidrapan** [reply]
Great! Thanks.
